class.jetpack-connection-banner.php

```diff
@@ -98,6 +84,8 @@ function maybe_initialize_hooks( $current_screen ) {
 			return;
 		}
 
+		$this->logo = new Logo();
```

Comment:
FWIW, if we have 10 usages of `Logo` throughout the plugin, we'll have 10 different `Logo` instances. Is that a bad or a good thing? My thoughts:

* Compared to what Michael suggested, we have 9 more equal objects. In this case, we don't need that, so the singleton was a good idea for a package that small.
* If this was a big package (sync, connection) we likely wouldn't want to initialize a bunch of things twice or more times. SIngleton comes in handy with that.